SConstruct

```diff
@@ -418,10 +418,18 @@ if not GetOption('clean') and not GetOption('help'):
 # ==============================compile==============================
 #
 env.Object(slmsource)
-env.Command('src/pinyin/quanpin_trie.h', 'python/quanpin_trie_gen.py',
-            'cd ${SOURCE.dir} && ./quanpin_trie_gen.py > ../src/pinyin/quanpin_trie.h')
-env.Command('src/pinyin/pinyin_info.h', 'python/pinyin_info_gen.py',
-            'cd ${SOURCE.dir} && ./pinyin_info_gen.py > ../src/pinyin/pinyin_info.h')
+if GetOS() == 'Darwin':
```

Comment:
@Inokinoki i think a better way to address this issue is not to rely on the `$PATH` env variable, but to provide a machinery to allow user to specify the path to the python3 interpreter. i created #117. could you help test it instead?